variants/backend-base/app/template.rb

```diff
@@ -19,4 +19,4 @@
 # Configure the default mailer to use the our default from address
 gsub_file "app/mailers/application_mailer.rb",
           "default from: 'from@example.com'",
-          "default from: Rails.application.secrets.mail_from"
+          "default from: Rails.application.config.app.mail_from"
```

Comment:
We cannot load things directly in to `Rails.application.config` using a YAML file - we have to pick a namespace under `config`. I chose `config.app` but totally open to changing this. 

Alternatively we could skip the YAML file completely and just have a ruby initializer which loads things directly into `Rails.application.config`. I slightly favour the YAML file because it makes it easier when secrets vary between environments.